wire-library/wire-runtime-swift/src/main/swift/wellknowntypes/Timestamp.swift

```diff
@@ -131,10 +130,38 @@ extension Timestamp : Proto3Codable {
 
 #if !WIRE_REMOVE_CODABLE
 extension Timestamp : Codable {
-    public enum CodingKeys : String, CodingKey {
+    public init(from decoder: Decoder) throws {
```

Comment:
@efirestone this is also incorrect:

proto3 | JSON | JSON example | Notes
-- | -- | -- | --
Timestamp | string | "1972-01-01T10:00:20.021Z" | Uses RFC 3339, where generated output will always be Z-normalized and uses 0, 3, 6 or 9 fractional digits. Offsets other than "Z" are also accepted.
Duration | string | "1.000340012s", "1s" | Generated output always contains 0, 3, 6, or 9 fractional digits, depending on required precision, followed by the suffix "s". Accepted are any fractional digits (also none) as long as they fit into nano-seconds precision and the suffix "s" is required.